### PR TITLE
add multi language option in settings, save choice in storage

### PR DESCRIPTION
**Pull request by Maximbndrnk**
image

image

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <name>iFarm</name>
     <description>An awesome Ionic/Cordova app.</description>
     <author email="hi@ionicframework" href="http://ionicframework.com/">Ionic Framework Team</author>
-    <content original-src="index.html" src="http://192.168.1.103:8100" />
+    <content src="index.html" />
```
Comment (dularion) [inline config.xml:6]: what is  the deal with this commit?

File: config.xml [hunk 2]
```diff
@@ -89,4 +89,5 @@
         <variable name="UPDATE_METHOD" value="auto" />
     </plugin>
     <engine name="android" spec="7.0.0" />
+    <plugin name="cordova-sqlite-storage" spec="3.1.0" />
```
Comment (dularion) [inline config.xml:92]: why use sqlite for something as tiny as the language selection? i think localstorage works well too, dont u think?

File: src/pages/list/list.ts [hunk 3]
```diff
@@ -19,13 +27,21 @@ export class ListPage {
     'american-football', 'boat', 'bluetooth', 'build'];
 
     this.items = [];
-    for (let i = 1; i < 11; i++) {
+    for (let i = 1; i < 5; i++) {
```
Comment (dularion) [inline src/pages/list/list.ts:30]: why change this? this is anyway unused.. isnt it?

**Review by dularion — CHANGES_REQUESTED** [review]

File: src/pages/list/list.ts (hunk 3 above)
Comment (Maximbndrnk) [reply src/pages/list/list.ts:30]: it is, I reduced it because it takes all screen

File: config.xml (hunk 2 above)
Comment (Maximbndrnk) [reply config.xml:92]: https://ionicframework.com/docs/building/storage - following this guide. I think better to use native stuff

File: config.xml (hunk 1 above)
Comment (Maximbndrnk) [reply config.xml:6]: Don't know, I do nothing, auto changed.

File: config.xml (hunk 2 above)
Comment (dularion) [reply config.xml:92]: ok good point

File: config.xml (hunk 1 above)
Comment (dularion) [reply config.xml:6]: its better not to commit what you didnt code on purpose ..

Comment (dularion) [reply config.xml:6]: but leave it for now. but next time look over every code change during a commit and only commit stuff that your purposefully put there

**Review by dularion — APPROVED** [review]

**Pull request merged**